index.html

```diff
@@ -0,0 +1,221 @@
+    ></script>
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <title>Capstone Project1</title>
+  </head>
+  <body>
+    <header class="desktop-nav-header">
+      <div class="top-nav">
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#" class="top-nav-link"> English </a>
+        <a href="#" class="top-nav-link"> myPage </a>
+        <a href="#" class="top-nav-link last-item"> logout </a>
+      </div>
+      <nav class="desktop">
+        <div class="desktop-logo">
+          <a href="index.html" class="desk-logo">
+            <img src="images/logo.png" alt="logo" />
+          </a>
+        </div>
+        <div class="desktop-nav">
+          <ul class="desktop-nav-lists">
+            <li class="nav-list">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#program">Program</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#speakers">Speakers</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#Sponsors">Sponsor</a>
+            </li>
+            <li class="nav-list">
+              <a href="">
+                <button class="nav-btn btn">Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Hero section -->
+    <section class="hero" id="home">
+      <button class="nav-toggle" aria-label="toggle-navigation">
+        <!-- <span class="hamburger"></span> -->
+        <div class="hamburger"></div>
+      </button>
+      <nav class="mobile-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">about</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">Campaign</button>
+        </a>
+      </nav>
+      <div class="hero-body">
+        <h2 class="hero-med-title">"Hello! Rugby fans"</h2>
+        <h1 class="hero-main-title">
+          Welcome to the home of Cape Town Rugby 10's
+        </h1>
+        <p class="hero-subtitle">
+          A fun festival where people from all around the world come together to
+          support their teams.
+        </p>
+        <h3 class="venue">3 – 4 FEBRUARY 2023 | ENTRIES OPEN 1 APRIL 2022</h3>
+      </div>
```

Comment:
- [ ] I honestly appreciate the way you customized the design of your website. Kudos  🦾.The text in your desktop view homepage is currently not readable due to the background and the color combination which will be difficult for users with eye impairment to use your website 👍🏽.


In a more professional working environment, it is very important to always stick to design as much as possible. In your case, you didn't 100% follow the design guideline given to you by microverse in your entire website, so I would recommend you go back to the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and follow it, starting from the background, typography, color, and others. This will make your website stand out and help you in a real-life scenario 😄 . good luck 👍🏽 


| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190243010-0ca3d5c2-4886-4a1a-8ef9-be62d67f73ff.png)  | ![image](https://user-images.githubusercontent.com/63473584/190243267-5cff979d-9ef7-4442-a9c2-7ba946ba1d6c.png)  |
| ![image](https://user-images.githubusercontent.com/63473584/190243138-adbfbc8e-d734-4c32-99a0-c2c305bb0127.png)  | ![image](https://user-images.githubusercontent.com/63473584/190243401-54c1fb6a-26ab-47de-ada4-33ae2fa0f1f8.png)  |